Tick2q/simulate.fs

```diff
@@ -52,12 +54,23 @@
         (envt, envt) ||> Map.fold (fun (env: Environment) (vName:string) (var:Variable) -> 
             updateEnvt vName (var.UpdateFn envt) env)
 
-    /// return environments after 0, 1, ... n applications of the step function
+    /// Return environments after 0, 1, ... n applications of the step function
     let nSteps (envt:Environment) (n:uint32): Environment list =
         (envt, [1u..n]) 
         ||> List.scan (fun env _i -> step env)
 
 
+    /// Logic Invertor function
+    // Given here as example. Use the individual update functions to produce
+    // a single correct one.
+    let wInvert (f1:Update) : Update =
+        fun env ->
+            f1 env
+            |> Result.map (function | Zero -> One | One -> Zero)
+        
+    /// Ok constant logic value
+    let wConst (w: Wire): Update =
+        failwithf "wconst not implemented"
```

Comment:
Moved this up from below as I actually finished Tick2 without realising an example had been done for me. Would have saved me a lot of time had I seen it upfront.

The test will also pass without the implementation of `wConst`, so moving it up increases the chance that a student would actually implement it.